src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3139,10 +3139,10 @@
     <value>Properties</value>
   </data>
   <data name="OpenClassicProperties" xml:space="preserve">
-    <value>Open File Explorer properties</value>
+    <value>File Explorer properties</value>
   </data>
   <data name="OpenPropertiesDescription" xml:space="preserve">
-    <value>Open properties window</value>
+    <value>Open the properties window for selected item</value>
```

Comment:
> for selected item

It's a little more complex. This action also works when no items are selected, and it also works when multiple items are selected. We can make the text contextual in the future, but it's out of scope for this PR. For now, I would keep the added 'the' but remove 'for selected item'